std/xml.d

```diff
@@ -1059,9 +1059,10 @@ class Tag
                 munch(s,whitespace);
                 reqc(s,'=');
                 munch(s,whitespace);
-                reqc(s,'"');
-                string val = decode(munch(s,"^\""), DecodeMode.LOOSE);
-                reqc(s,'"');
+                string atr_val_put_chars="'\"";   //chars in which tags can be put into
+                reqs(s,atr_val_put_chars);
+                string val = decode(munch(s,"^'\""), DecodeMode.LOOSE);
+                reqs(s,atr_val_put_chars);
```

Comment:
This behaves incorrectly for stuff `foo="bar 'baz' qux"`. You have to check which quote is used in the beginning, and stop when hitting it again.